### PR TITLE
Feed enrichment exclusion

### PR DESCRIPTION
**Pull request by itssapir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-11809](https://jira-dc.paloaltonetworks.com/browse/CIAC-11809)

## Description
Adds the "Exclude Enrichment" param to the following feeds:

- CSV Feed
- Elasticsearch Feed
- Github Feed
- JSON Feed
- Plain text feed
- RSS feed
- TAXII feed

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bziser — APPROVED** [review]

**Comment by itssapir** [comment]
Fails on:
- TAXII_Feed_Test (Mock Disabled) - [Ticket](https://jira-dc.paloaltonetworks.com/browse/CIAC-10573)
- TAXII2 Server Performance Test (Mock Disabled) - [Ticket](https://jira-dc.paloaltonetworks.com/browse/CIAC-9768)

File: Packs/BitcoinAbuse/ReleaseNotes/1_0_43.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### BitcoinAbuse Feed (Deprecated)
+
+- Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```
Comment (ShirleyDenkberg) [inline Packs/BitcoinAbuse/ReleaseNotes/1_0_43.md:6]:
```suggestion
Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```

File: Packs/CommonScripts/ReleaseNotes/1_15_65.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### StixCreator
+
+- Enhanced **TAXII2FeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this script.
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/ReleaseNotes/1_15_65.md:6]:
```suggestion
Enhanced **TAXII2FeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this script.
```

File: Packs/FeedAlienVault/ReleaseNotes/1_1_33.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### AlienVault Reputation Feed
+
+- Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```
Comment (ShirleyDenkberg) [inline Packs/FeedAlienVault/ReleaseNotes/1_1_33.md:6]:
```suggestion
Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```

File: Packs/FeedBambenekConsulting/ReleaseNotes/1_2_4.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Bambenek Consulting Feed
+
+- Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```
Comment (ShirleyDenkberg) [inline Packs/FeedBambenekConsulting/ReleaseNotes/1_2_4.md:6]:
```suggestion
Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```

File: Packs/FeedDHS/ReleaseNotes/2_0_43.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### DHS Feed v2
+
+- Enhanced **TAXII2FeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```
Comment (ShirleyDenkberg) [inline Packs/FeedDHS/ReleaseNotes/2_0_43.md:6]:
```suggestion
Enhanced **TAXII2FeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```

File: Packs/FeedElasticsearch/ReleaseNotes/1_1_8.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Elasticsearch Feed
+
+<~XSOAR_SAAS>
+- Support for instance-wide indicator enrichment exclusion.
```
Comment (ShirleyDenkberg) [inline Packs/FeedElasticsearch/ReleaseNotes/1_1_8.md:7]:
```suggestion
Added support for instance-wide indicator enrichment exclusion.
```

File: Packs/FeedMajesticMillion/ReleaseNotes/1_1_16.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Majestic Million Feed
+
+- Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```
Comment (ShirleyDenkberg) [inline Packs/FeedMajesticMillion/ReleaseNotes/1_1_16.md:6]:
```suggestion
Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```

File: Packs/FeedMitreAttackv2/ReleaseNotes/1_1_42.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### MITRE ATT&CK
+
+- Enhanced **TAXII2FeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```
Comment (ShirleyDenkberg) [inline Packs/FeedMitreAttackv2/ReleaseNotes/1_1_42.md:6]:
```suggestion
Enhanced **TAXII2FeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```

File: Packs/FeedGitHub/ReleaseNotes/1_0_10.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Github Feed
+- Updated the Docker image to: *demisto/taxii2:1.0.0.112358*.
+
+
+<~XSOAR_SAAS>
+- Support for instance-wide indicator enrichment exclusion.
```
Comment (ShirleyDenkberg) [inline Packs/FeedGitHub/ReleaseNotes/1_0_10.md:9]:
```suggestion
- Added support for instance-wide indicator enrichment exclusion.
```

File: Packs/FeedCSV/ReleaseNotes/1_1_32.md [hunk 10]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### CSV Feed
+- Updated the Docker image to: *demisto/py3-tools:1.0.0.112434*.
+
+
+<~XSOAR_SAAS>
+- Support for instance-wide indicator enrichment exclusion.
```
Comment (ShirleyDenkberg) [inline Packs/FeedCSV/ReleaseNotes/1_1_32.md:9]:
```suggestion
- Added support for instance-wide indicator enrichment exclusion.
```

File: Packs/FeedJSON/ReleaseNotes/1_1_32.md [hunk 11]
```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+
+##### JSON Feed
+- Updated the Docker image to: *demisto/py3-tools:1.0.0.112434*.
+
+<~XSOAR_SAAS>
+- Support for instance-wide indicator enrichment exclusion.
```
Comment (ShirleyDenkberg) [inline Packs/FeedJSON/ReleaseNotes/1_1_32.md:8]:
```suggestion
- |Added support for instance-wide indicator enrichment exclusion.
```

File: Packs/FeedPlainText/ReleaseNotes/1_1_28.md [hunk 12]
```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+
+##### Plain Text Feed
+- Updated the Docker image to: *demisto/py3-tools:1.0.0.112434*.
+
+<~XSOAR_SAAS>
+- Support for instance-wide indicator enrichment exclusion.
```
Comment (ShirleyDenkberg) [inline Packs/FeedPlainText/ReleaseNotes/1_1_28.md:8]:
```suggestion
- Added support for instance-wide indicator enrichment exclusion.
```

File: Packs/FeedRSS/ReleaseNotes/1_0_11.md [hunk 13]
```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+
+##### RSS Feed
+- Updated the Docker image to: *demisto/py3-tools:1.0.0.112434*.
+
+<~XSOAR_SAAS>
+- Support for instance-wide indicator enrichment exclusion.
```
Comment (ShirleyDenkberg) [inline Packs/FeedRSS/ReleaseNotes/1_0_11.md:8]:
```suggestion
- Added support for instance-wide indicator enrichment exclusion.
```

File: Packs/FeedTAXII/ReleaseNotes/1_2_20.md [hunk 14]
```diff
@@ -0,0 +1,15 @@
...
+
+<~XSOAR_SAAS>
+- Support for instance-wide indicator enrichment exclusion.
+</~XSOAR_SAAS>
+
+##### TAXII 2 Feed
+- Updated the Docker image to: *demisto/taxii2:1.0.0.112358*.
+
+<~XSOAR_SAAS>
+- Support for instance-wide indicator enrichment exclusion.
```
Comment (ShirleyDenkberg) [inline Packs/FeedTAXII/ReleaseNotes/1_2_20.md:14]:
```suggestion
Added support for instance-wide indicator enrichment exclusion.
</~XSOAR_SAAS>

##### TAXII 2 Feed
- Updated the Docker image to: *demisto/taxii2:1.0.0.112358*.

<~XSOAR_SAAS>
- Added support for instance-wide indicator enrichment exclusion.
```

File: Packs/FeedURLhaus/ReleaseNotes/1_0_12.md [hunk 15]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### URLhaus Feed
+
+- Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```
Comment (ShirleyDenkberg) [inline Packs/FeedURLhaus/ReleaseNotes/1_0_12.md:6]:
```suggestion
Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```

File: Packs/FeedUnit42v2/ReleaseNotes/1_0_58.md [hunk 16]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Unit 42 ATOMs Feed
+
+- Enhanced **TAXII2FeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```
Comment (ShirleyDenkberg) [inline Packs/FeedUnit42v2/ReleaseNotes/1_0_58.md:6]:
```suggestion
Enhanced **TAXII2FeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```

File: Packs/FeedZeroDayLiveTIFusion/ReleaseNotes/1_0_5.md [hunk 17]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Zero Day Live TI FUSION Feed
+
+- Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```
Comment (ShirleyDenkberg) [inline Packs/FeedZeroDayLiveTIFusion/ReleaseNotes/1_0_5.md:6]:
```suggestion
Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```

File: Packs/Feedsslabusech/ReleaseNotes/1_1_28.md [hunk 18]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### abuse.ch SSL Blacklist Feed
+
+- Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```
Comment (ShirleyDenkberg) [inline Packs/Feedsslabusech/ReleaseNotes/1_1_28.md:6]:
```suggestion
Enhanced **CSVFeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```

File: Packs/TAXIIServer/ReleaseNotes/2_0_67.md [hunk 19]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### TAXII2 Server
+
+- Enhanced **TAXII2FeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/ReleaseNotes/2_0_67.md:6]:
```suggestion
Enhanced **TAXII2FeedApiModule** to support setting indicator enrichment exclusion in select feeds. The change has no impact on this integration.
```

**Comment by ShirleyDenkberg** [comment]
@bziser @MLainer1 @Ni-Knight Doc review completed.

**Review by Ni-Knight — APPROVED** [review]
Looks good, why is a force merge needed?

**Pull request merged**